### DISCUSSION TITLE
Adding a default create-* script for organizations for npm_init

### DISCUSSION THREAD

**Discussion by overthemike** [npm/feedback#486] [CLI] [answered]
If an initializer is used for an organization, e.g. `npm init @someorg/scaffold`, it would be very cool to have a fallback default script that can be called if it exists if that particular package doesn't exist. It could allow for dynamic scaffolding or even something as simple as a `did you mean "*"`. Org owners could add a `create-default` or `default-init` (or something similar) package that would be called if the initializer that was tried doesn't exist. Could also be useful for just a default script to be called per org by not calling anything, like `npm init @someorg`.

**Comment by MylesBorins** [comment, marked as answer]
hey @overthemike. The next step to try and build consensus around a change like this will be to open an RFC issue with the CLI team

https://github.com/npm/rfcs
